src/Features/Onboarding.php

```diff
@@ -281,16 +300,8 @@ public static function get_themes() {
 			$themes     = array();
 
 			if ( ! is_wp_error( $theme_data ) ) {
-				$theme_data = json_decode( $theme_data['body'] );
-				usort(
-					$theme_data->products,
-					function ( $product_1, $product_2 ) {
-						if ( 'Storefront' === $product_1->slug ) {
-							return -1;
-						}
-						return $product_1->id < $product_2->id ? 1 : -1;
-					}
-				);
+				$theme_data           = json_decode( $theme_data['body'] );
+				$theme_data->products = self::sort_woocommerce_themes( $theme_data->products );
```

Comment:
We should check for the `products` property first, eh?

Nitpick - no need to overwrite this property. Could just use a new variable.